Apps/Sandcastle/gallery/GeoJSON and TopoJSON.html

```diff
@@ -82,7 +82,7 @@
         }
     }).otherwise(function(error){
         //Display any errrors encountered while loading.
-        window.alert(error);
+        window.alert(error); //eslint-disable-line no-alert
```

Comment:
We should never use eslint pragmas for code that shows up in Sandcastle examples, instead ignore `no-alert` in `Apps/Sandcastle/.eslintrc.json`